### PR TITLE
enable windows build

### PR DESCRIPTION
**Pull request by kejxu**

### CODE REVIEW

File: ecl_config/include/ecl/config.hpp [hunk 1]
```diff
@@ -31,7 +31,7 @@
  ****************************************************************************/
 
 #if defined(ECL_IS_WIN32)
-	#include "config/windows.hpp"
+    // #include "config/windows.hpp"
```
Comment (ooeygui) [inline ecl_config/include/ecl/config.hpp]: Commented out code

**Review by ooeygui — APPROVED** [review]
Just remove the commented out code.

**Review by ooeygui — APPROVED** [review]

File: ecl_config/include/ecl/config/windows.hpp [hunk 2]
```diff
@@ -29,10 +29,10 @@
   #endif
 
   #ifdef _MSC_VER
-    #pragma warning(disable: 4251)  // Disable warnings about import/exports when deriving from std classes
-    #pragma warning(disable: 4275)  // ""
-    #pragma warning (disable:4996)  // Disable warnings about deprecated ctime
-    #pragma warning (disable:4290)  // Disable warnings about unsupported c++ exception specifications
+    #pragma warning(disable:4251)  // Disable warnings about import/exports when deriving from std classes
```
Comment (seanyen) [inline ecl_config/include/ecl/config/windows.hpp]: nit: I would recommend to drop the cosmetic fixes, so people won't get distracted by them.

**Review by seanyen — APPROVED** [review]
LGTM

File: ecl_config/include/ecl/config/windows.hpp (hunk 2 above)
Comment (kejxu) [reply ecl_config/include/ecl/config/windows.hpp]: fixed

**Pull request merged**